README.md

```diff
@@ -1,5 +1,9 @@
-# ![nf-core/phageannotator](docs/images/nf-core-phageannotator_logo_light.png#gh-light-mode-only) ![nf-core/phageannotator](docs/images/nf-core-phageannotator_logo_dark.png#gh-dark-mode-only)
-
+<h1>
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-phageannotator_logo_dark.png">
+    <img alt="nf-core/phageannotator" src="docs/images/nf-core-phageannotator_logo_light.png">
+  </picture>
+</h1>
```

Comment:
```suggestion
</h1>

```


I think you need a line break here. We just had messed up readme badges in a different pipeline. https://github.com/nf-core/sarek/pull/1390/files